generic_throttle/src/generic_throttle/generic_throttle.py

```diff
@@ -123,3 +153,32 @@ def _populate_dictionary(self):
 
     def _shutdown(self):
         rospy.loginfo('Stopping ' + str(rospy.get_name()))
+
+    def _resize_image(self, data, resolution):
+        old_image = None
+
+        try:
+            old_image = self.bridge.imgmsg_to_cv2(data)
+        except CvBridgeError as e:
+            print(e)
+            exit(20)
+
+        factor = 1.0 / resolution
```

Comment:
if the parameter is now called factor already then there should not be another 1/factor...it makes it confusing...
if you want the resolution to be halfed...you should set the resolution_factor param to 0.5 in the yaml...and not to 2!